### PR TITLE
Gnome package updates

### PR DESCRIPTION
**Pull request by satmandu**
Builds properly:
- [x] `x86_64`
- [x] `armv7l` 

### Run the following to get this pull request's changes locally for testing.
```
CREW_REPO=https://github.com/satmandu/chromebrew.git CREW_BRANCH=gnome crew update
```

### CODE REVIEW

File: packages/py3_pylint.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+require 'buildsystems/pip'
+
+class Py3_pylint < Pip
+  description 'Analyzes Python code looking for bugs and signs of poor quality'
+  homepage 'https://pylint.pycqa.org'
+  version '3.1.0'
+  license 'GPL'
+  compatibility 'all'
+  source_url 'SKIP'
+  binary_compression 'tar.zst'
```
Comment (Zopolis4) [inline packages/py3_pylint.rb:13]: We can probably drop the binary_compression if there arent any binaries.

File: packages/libchamplain.rb [hunk 2]
```diff
@@ -3,24 +3,34 @@
 class Libchamplain < Meson
   description 'A map widget'
   homepage 'https://wiki.gnome.org/Projects/libchamplain'
-  version '0.12.20'
+  version '0.12.21'
```
Comment (uberhacker) [inline packages/libchamplain.rb:6]: No filelist updates?

File: packages/fuse3.rb [hunk 3]
```diff
@@ -1,36 +1,30 @@
-require 'package'
+require 'buildsystems/meson'
 
-class Fuse3 < Package
+class Fuse3 < Meson
   description 'The reference implementation of the Linux FUSE (Filesystem in Userspace) interface.'
   homepage 'https://github.com/libfuse/libfuse/'
-  version '3.14.0'
+  version '3.16.2'
```
Comment (uberhacker) [inline packages/fuse3.rb:6]: No i686 filelist updates?

File: packages/gvfs.rb [hunk 4]
```diff
@@ -3,7 +3,7 @@
 class Gvfs < Meson
   description 'Virtual filesystem implementation for GIO'
   homepage 'https://wiki.gnome.org/Projects/gvfs'
-  version '1.50.4'
+  version '1.54.0'
```
Comment (uberhacker) [inline packages/gvfs.rb:6]: No binary updates?

Comment (uberhacker) [inline packages/gvfs.rb:6]: No binary updates?

File: packages/py3_pylint.rb [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+# https://gitlab.archlinux.org/archlinux/packaging/packages/python-pylint/-/blob/main/PKGBUILD?ref_type=heads
+
+require 'buildsystems/pip'
+
+class Py3_pylint < Pip
+  description 'Analyzes Python code looking for bugs and signs of poor quality'
+  homepage 'https://pylint.pycqa.org'
+  version '3.1.0'
+  license 'GPL'
+  compatibility 'all'
```
Comment (uberhacker) [inline packages/py3_pylint.rb:11]: No i686 filelist updates?

File: packages/libchamplain.rb (hunk 2 above)
Comment (satmandu) [reply packages/libchamplain.rb:6]: Minor version update. No file list changes.

File: packages/fuse3.rb (hunk 3 above)
Comment (satmandu) [reply packages/fuse3.rb:6]: No change

File: packages/gvfs.rb (hunk 4 above)
Comment (satmandu) [reply packages/gvfs.rb:6]: Fixed

**Review by uberhacker — APPROVED** [review]

**Pull request merged**